ocaml-lsp-server/src/diagnostics.ml

```diff
@@ -0,0 +1,64 @@
+open Import
+
+module Uri = struct
+  include Uri
+
+  let compare x y = Ordering.of_int (Uri.compare x y)
```

Comment:
Where does `uri` come from? Can we upstream this implementation of `compare`?

Reply:
We cannot because `Ordering` is from stdune and we are trying to avoid using stdune in lsp.